### DISCUSSION TITLE
How to get the editor to parse custom tags?

### DISCUSSION THREAD

**Discussion by ghost** [ueberdosis/tiptap#1741] [General Questions] [answered]
Hi, all - new TipTap user here. I have been trying to get the editor to parse a custom tag in its content ("&lt;live&gt;"). All the examples I've seen make this look straightforward, but I cannot get it to work. I suspect there is something very basic that I just haven't grasped about how all this works.

Here is my code. I'm expecting to see the output of editor.getHTML() look like this:

_&lt;p&gt;This should be parsed as a LiveTag: &lt;live&gt;foo&lt;/live&gt;&lt;p&gt;_

But instead the &lt;live&gt; tags are stripped out and it just looks like this:

_&lt;p&gt;This should be parsed as a LiveTag: foo&lt;p&gt;_

Can anyone see what I'm doing wrong? Thanks so much!

```
<template>
  <div>
    <editor-content :editor="editor" />
    <div>{{ editor ? editor.getHTML() : "" }}</div>
  </div>
</template>

<script>
import { Editor, EditorContent } from "@tiptap/vue-2";
import StarterKit from "@tiptap/starter-kit";
import { Node, mergeAttributes } from "@tiptap/core";

let LiveTag = Node.create({
  name: "live",
  content: "text*",
  defaultOptions: {
    HTMLAttributes: {},
  },
  parseHTML() {
    return [
      {
        tag: "live",
      },
    ]
  },
  renderHTML({ HTMLAttributes }) {
    return ["live", mergeAttributes(HTMLAttributes, {}), 0]
  },
});

export default {
  props: ["value"],

  components: {
    EditorContent,
  },

  data() {
    return {
      editor: null,
    }
  },

  mounted() {
    let starting_value = `<p>This should be parsed as a LiveTag: <live>foo</live>`;
    this.editor = new Editor({
      extensions: [
        LiveTag,
        StarterKit,
      ],
      content: starting_value,
    });
  },

  beforeDestroy() {
    this.editor.destroy()
  },
}
</script>

```

**Comment by laelliang** [comment, marked as answer]
参考 https://www.tiptap.dev/examples/interactivity 定义 live 组件，然后在示例“Extension.js”里的Node.create加上  group: 'inline',inline: true,
然后就可以正常显示了

**Reply by philippkuehn** [reply]
@liangchengen Please english only so everyone can understand your message.

@Dausuul You have to define an inline node for that.

```diff
let LiveTag = Node.create({
  //...
+  group: 'inline',
+  inline: true,
  //...
})

```

**Reply by ghost** [reply]
It works! Many thanks to both of you for the quick answer.
